src/styles/components/_toc.scss

```diff
@@ -0,0 +1,161 @@
+@import '../../styles/settings/colors';
+@import '../../styles/settings/dimensions';
+@import '../../styles/settings/type';
+@import '../../styles/tools/breakpoints';
+@import '../../styles/tools/mixins';
+
+@mixin TOC_LINKS {
+  @include bp(md) {
+    color: $PRIMARY_TEXT_COLOR;
+
+    &.w-toc__active--link,
+    &:hover {
+      color: $WEB_PRIMARY_COLOR;
+    }
+  }
+}
+
+@mixin TOC_PADDING {
+  padding: 0 24px;
+}
+
+.w-toc__active--border {
```

Comment:
I think you're reaching for a BEM modifier when that's not the correct syntax to use. It looks like you're hitting the "grandchild selector" issue https://www.smashingmagazine.com/2016/06/battling-bem-extended-edition-common-problems-and-how-to-avoid-them/

If the border is just an element inside of the toc block then it's fine to do:

```
.w-toc__border
```